shared/syncs.schema.json

```diff
@@ -169,7 +169,7 @@
                 "type": { "const": "parent_children" },
                 "parent": { "$ref": "#/definitions/parentInfo" },
                 "parentFieldlist": { "$ref": "#/definitions/fieldlist" },
-                "children": { "$ref": "#/definitions/parentInfo" },
+                "children": { "$ref": "#/definitions/childrenInfo" },
```

Comment:
This one went unnoticed because childrenInfo contains all the field in parentInfo, therefore a valid json will pass validation when checking children against the schema for parentInfo.

Reply:
Looks like the same fix is needed for "parentChildrenSyncUpTarget".